frontend/src/views/web/challenge/leaderboard.html

```diff
@@ -28,28 +28,54 @@
                         </div>
                     </div>
                 </div>
-                <div class=" ev-card-body exist-team-card">
+                <div class="ev-card-body exist-team-card">
                     <div class="row">
                         <div class="col s12">
                             <div ng-if="!challenge.isResult" class="result-wrn">No phase selected.</div>
                             <table ng-if="challenge.isResult && challenge.leaderboard.length" class="highlight">
                                 <thead>
-                                   <tr>
-                                        <td class="align-left" data-field="rank">Rank</td>
-                                        <td class="align-left" data-field="team">Participant Team</td>
-                                        <td class="align-right" ng-repeat="key in challenge.leaderboard[0].leaderboard__schema.labels">{{key}}</td>
-                                        <td class="align-right" data-field="submission_time">Last Submission at</td>
-                                    </tr>
+                                <tr>
+                                    <td class="align-left" data-field="rank">
```

Comment:
It looks like you are not shifting the value of Rank column when sorted. Can you please make sure that this column value is also shifted whenever someone tries to sort the leaderboard?

Reply:
![final](https://user-images.githubusercontent.com/8584694/38247286-ff930412-3744-11e8-9b08-e9cb93bbda12.png)

Does it look good?

Remark: values in rank column will be always 1 ... n or n ... 1. In other words, sorting does not preserve the initial ranking number of a participant team. Should it be like this or you want to keep the initial values when the leaderboard is sorted by some other column?

Reply:
another question: do the arrows look good or I should reverse them?

Reply:
> Remark: values in the rank column will be always 1 ... n or n ... 1. In other words, sorting does not preserve the initial ranking number of a participant team. Should it be like this or you want to keep the initial values when the leaderboard is sorted by some other column?

The rank of the participants should be preserved to keep the initial values when the leaderboard is sorted by some other column.

Also, it would be even cooler if we can add the feature to have a unique URL for each sorting type.
For example: 
Sort in ascending order by `score1` has URL: `http://localhost:8888/web/challenges/challenge-page/1/leaderboard/1?sortby=score1`

Sort in descending order by `score1` has URL: `http://localhost:8888/web/challenges/challenge-page/1/leaderboard/1?sort=-score1`

Let me know if you will be able to add this feature in this pull request. Otherwise, we can open a new issue for this.

Reply:
> another question: do the arrows look good or I should reverse them?

I do think that the arrows can look much better. Can you check this table UI: https://dribbble.com/shots/4218286-Media-Configuration-Table

We would like to have something similar where the column arrow gets highlighted when it is sorted by that. Do you see what I mean?

Reply:
Is there a way to have these URLs without modifying the backend code? If there is one, can you give me an example? If no, then I should be able to do this by modifying challenge urls.

Regarding the arrows: consider it done.

Reply:
> Is there a way to have these URLs without modifying the backend code? If there is one, can you give me an example? If no, then I should be able to do this by modifying challenge urls.

I think it's fine for now to not have the URL feature. That would delay this feature. We would go with the current implementation only once you have changed the arrows.